NeoML/src/TraditionalML/GradientBoost.cpp

```diff
@@ -252,19 +252,20 @@ CPtr<IGradientBoostRegressionModel> CGradientBoost::TrainRegression(
 
 CPtr<IRegressionModel> CGradientBoost::TrainRegression( const IRegressionProblem& problem )
 {
-	if( logStream != 0 ) {
+	if( logStream != nullptr ) {
 		*logStream << "\nGradient boost regression training started:\n";
 	}
 
 	CPtr<const IMultivariateRegressionProblem> multivariate =
-		FINE_DEBUG_NEW CMultivariateRegressionOverUnivariate( dynamic_cast<const IRegressionProblem*>( &problem ) );
+		FINE_DEBUG_NEW CMultivariateRegressionOverUnivariate( 
+			dynamic_cast<const IRegressionProblem*>( &problem ) );
```

Comment:
dynamic_cast?..

Reply:
this code is not mine, just added a newline

Reply:
yep, it's redundant. removed now